doc-content/enterprise-only/vehicle-route-planning/vrp-custom-data-sets.adoc

```diff
@@ -0,0 +1,25 @@
+[id='vrp-custom-data-sets_{context}']
+
+= Creating custom data sets with OptaWeb Vehicle Routing
+
+There is a built-in demo data set consisting of a several large Belgian cities.
+If you want to have more demos available in the *Load demo* menu, you can prepare your own data sets.
+
+.Procedure
+To do that, follow these steps:
+
+. In OptaWeb Vehicle Routing, add a depot and a number of visits by clicking on the map or using geosearch.
+. Click *Export* and save the file in the data set_directory.
++
+[NOTE]
+====
+The data set directory is the directory specified in the `app.demo.data-set-dir` property.
+
+If the application is running through the `runLocally.sh` script, the data set directory is set to `$HOME/{data-dir-name}/dataset`.
+
+Otherwise, the property is from `application.properties` and defaults to `optaweb-vehicle-routing-backend/local/dataset`.
```

Comment:
That is not correct.
Basically the VRP, in this case, checks local/dataset but in the directory where VHP have been launched
So Currently if user run it locally using jar file
optaweb-vehicle-routing-distribution-7.39.0.Final-redhat-00001/sources/optaweb-vehicle-routing-standalone/target/local/dataset instead of optaweb-vehicle-routing-backend/local/dataset
Also it probably worth to mention that this could be setup with property app.demo.data-set-dir

Reply:
Updated, please confirm